src/main/java/ch/wisv/events/model/Event.java

```diff
@@ -106,24 +110,22 @@ public void setRegistrationLimit(int registrationLimit) {
         return registrations;
     }
 
+    public boolean isRegistrationOpen() {
+        LocalDateTime now = LocalDateTime.now();
+        return registrationStart.isBefore(now) && registrationEnd.isAfter(now);
+    }
+
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
         if (!(o instanceof Event)) return false;
         Event event = (Event) o;
-        return id == event.id &&
-                registrationLimit == event.registrationLimit &&
-                Objects.equals(title, event.title) &&
-                Objects.equals(description, event.description) &&
-                Objects.equals(start, event.start) &&
-                Objects.equals(end, event.end) &&
-                Objects.equals(registrationStart, event.registrationStart) &&
-                Objects.equals(registrationEnd, event.registrationEnd);
+        return id == event.id;
```

Comment:
I didn't test whether or not this was necessary actually. Figured the update wouldn't work otherwise, but that probably uses the `id` field instead of these functions.